workstation-backend/component/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from workspace import models as workspace_models
+
+# Create your models here.
+class BaseComponent(models.Model):
+    workspace_id = models.ForeignKey(workspace_models.Workspace, on_delete=models.CASCADE)
+    x = models.DecimalField(max_digits=100, decimal_places=2)
+    y = models.DecimalField(max_digits=100, decimal_places=2)
+    width = models.DecimalField(max_digits=100, decimal_places=2)
+    height = models.DecimalField(max_digits=100, decimal_places=2)
+
+    class Meta:
+        abstract = True
+
+
+# Checklist component
+class Checklist(BaseComponent):
+    title = models.CharField(max_length=200)
+    collapsed = models.BooleanField()
```

Comment:
We can put this as part of the abstract class. I think images should be collapsible too